--- project/app/sw.js
+++ project/app/sw.js
@@ -20,7 +20,21 @@ if (workbox) {
   console.log(`Yay! Workbox is loaded 🎉`);
   workbox.precaching.precacheAndRoute([]);
 
-  const bgSyncPlugin = new workbox.backgroundSync.Plugin('dashboardr-queue');
+  const showNotification = () => {
+    self.registration.showNotification('Background sync success!', {
+      body: '🎉`🎉`🎉`'
+    });
+  };
+  
+  const bgSyncPlugin = new workbox.backgroundSync.Plugin(
+    'dashboardr-queue',
+    {
+      callbacks: {
+        queueDidReplay: showNotification
+        // other types of callbacks could go here
+      }
+    }
+  );
 
   const networkWithBackgroundSync = new workbox.strategies.NetworkOnly({
     plugins: [bgSyncPlugin],
