diff --git a/project/app/js/main.js b/project/app/js/main.js
index fd1a289..08981f8 100644
--- a/project/app/js/main.js
+++ b/project/app/js/main.js
@@ -24,6 +24,17 @@ if ('serviceWorker' in navigator) {
         console.log(`Service Worker registration failed: ${err}`);
       });
   });
+
+  let deferredPrompt;
+
+  window.addEventListener('beforeinstallprompt', e => {
+    console.log('beforeInstallPrompt event detected');
+    // Prevent Chrome 76 and later from showing the mini-infobar
+    e.preventDefault();
+    // Stash the event so it can be triggered later.
+    deferredPrompt = e;
+    console.log('ready for A2HS');
+  });
 }
 
 function createIndexedDB() {
