diff --git a/project/app/sw.js b/project/app/sw.js
index 9e4eb08..9416ba1 100644
--- a/project/app/sw.js
+++ b/project/app/sw.js
@@ -19,6 +19,18 @@ importScripts('https://storage.googleapis.com/workbox-cdn/releases/5.0.0/workbox
 if (workbox) {
   console.log(`Yay! Workbox is loaded 🎉`);
   workbox.precaching.precacheAndRoute([]);
+
+  const { BackgroundSyncPlugin } = workbox.backgroundSync;
+  const { registerRoute } = workbox.routing;
+  const { NetworkOnly } = workbox.strategies;
+
+  const bgSyncPlugin = new BackgroundSyncPlugin('dashboardr-queue');
+
+  const networkWithBackgroundSync = new NetworkOnly({
+    plugins: [bgSyncPlugin]
+  });
+
+  registerRoute(/\/api\/add/, networkWithBackgroundSync, 'POST');
 } else {
   console.log(`Boo! Workbox didn't load 😬`);
 }
